nix/workbench/profile/prof1-variants.jq

```diff
@@ -276,7 +276,7 @@ def all_profile_variants:
    ({ generator:
       { plutus:
           { type:                       "LimitSaturationLoop"
-          , script:                     "v1/loop.plutus"
+          , script:                     { "Left": "Loop.hs" }
```

Comment:
I'd question whether the `.hs` suffix really belong in the script names.

We want the script name to be present in the reports -- and the report generator (and the rest of the workbench, by the way) mostly treats profile definitions as abstract, where all names are clean and usable directly.

If the report generator had to do the `.hs` suffix stripping, that'd be a pretty severe violation of the abstraction.

The same goes for the `Left`/`Right` wrappers -- they'd be unnatural to deal with.

So what I suggest -- since we're unlikely to use the Plutus scripts directly -- they all depend on the script-specific `tx-generator` knowledge to run them -- to just:

1. drop the `Left` wrappers from the profile definitions
2. unconditionally add the `Left` at the point where we pass the value to `tx-generator`

So it'd be a Nix-only change.